homeassistant/components/opnsense/__init__.py

```diff
@@ -36,44 +38,80 @@
     },
     extra=vol.ALLOW_EXTRA,
 )
+PLATFORMS = [Platform.DEVICE_TRACKER]
+
+
+@dataclass(slots=True)
+class OPNSenseDomainData:
+    """Dataclass to store interface clients."""
+
+    coordinators: dict[str, OPNSenseUpdateCoordinator]
```

Comment:
I think you want to do this to add some typing, Looking at other integration I don't think this adds value